console/types/src/lib.rs

```diff
@@ -16,8 +16,22 @@
 #![allow(clippy::too_many_arguments)]
 #![warn(clippy::cast_possible_truncation)]
 
+#[cfg(feature = "default")]
```

Comment:
Why are the `default` cfg flags necessary?

Reply:
`default` is necessary because the other dependent crates (like `console_types_address`, `console_types_integers`, etc) are made available under that flag.